src/julia.h

```diff
@@ -188,7 +188,8 @@ typedef struct _jl_sym_t {
     struct _jl_sym_t *left;
     struct _jl_sym_t *right;
     uptrint_t hash;    // precomputed hash value
-    JL_ATTRIBUTE_ALIGN_PTRSIZE(char name[]);
+    // variable length
+    JL_ATTRIBUTE_ALIGN_PTRSIZE(char name[1]);
```

Comment:
yep, same as all those other macros. fwiw, i'm not certain if it should be `jl_sym_t` or `jl_value_t` there, since lldb and C care about the exact pointer type, usually the src code sticks with jl_value_t to reduce the amount of casting.